packages/host/config/environment.js

```diff
@@ -40,6 +38,9 @@ module.exports = function (environment) {
     loginMessageTimeoutMs: 1000,
     minSaveTaskDurationMs: 1000,
     iconsURL: process.env.ICONS_URL || 'https://boxel-icons.boxel.ai',
+    stripePaymentLink:
+      process.env.STRIPE_PAYMENT_LINK ||
+      'https://buy.stripe.com/test_4gw01WfWb2c1dBm7sv',
```

Comment:
Having so many different `STRIPE_PAYMENT_LINK`s in multiple different places in this PR is a bit awkward, especially since we will also have the credit buy links too and this could quickly become unwieldy. I am wondering... since these links are a function of Stripe environment, and Stripe environment is dictated by the Stripe API key, why don't we have these links saved in the same place where API key lives, i.e. in the realm server? 

There could be an endpoint in the the realm server that responds with stripe payment links. That endpoint could fetch the active payment links (customer portal link, credit buy links) using Stripe API and return it to the host. That way we don't have to think about env vars for these links at all, and it removes any risk of misconfiguration across different deployments.

@FadhlanR i think you are working on similar things (credit buy links) so your thoughts are welcome

Reply:
I agree with adding an endpoint in the realm server for Stripe links. In my PR, I need to use the Stripe API key to fetch payment links and retrieve the URLs along with the credit_reload_amount. Currently, we only have three options for purchasing more credits, so storing them in the host's environment variables is manageable. However, if we add more options, it could become awkward to maintain multiple links in the environment variables. Having an endpoint would also make testing easier.

Reply:
My proposal for the API structure:

Route: `_payment-links`
Method: `GET`
Response:

```
{
    "data": [
        {
            "type": "payment-link",
            "id": "plink_1QP4pEPUHhctoJxaEp1D3myP",
            "data": {
                "url": "https://buy.stripe.com/test_bIY01W11heYNeFqbIO",
                "metadata": {
                    "credit_reload_amount": "15000"
                }
            }
        },
        {
            "type": "payment-link",
            "id": "plink_1QP4pEPUHhctoJxaEp1D3my!",
            "data": {
                "url": "https://buy.stripe.com/test_bIY01W11hesdfaeEasd",
                "metadata": {
                    "credit_reload_amount": "80000"
                }
            }
        }
    ]
}
```

Reply:
@lukemelia are you ok with this idea?

Reply:
@FadhlanR in your example you can also include the customer portal link, and the free plan payment link. That way we can define how the host app could differentiate between these types of stripe links.

Reply:
Route: _stripe-links
Method: GET
Response:

```
{
    "data": [
        {
            "type": "customer-portal-link",
            "id": "1",
            "attributes": {
                "url": "https://buy.stripe.com/test_bIY01W11heYNeFqbIH"
            }
        },
        {
            "type": "free-plan-payment-link",
            "id": "plink_1QP4pEPUHhctoJxaEp1D3myQ",
            "attributes": {
                "url": "https://billing.stripe.com/p/login/test_cN216h3BlbML3FS154"
            }
        },
        {
            "type": "extra-credits-payment-link",
            "id": "plink_1QP4pEPUHhctoJxaEp1D3myP",
            "attributes": {
                "url": "https://buy.stripe.com/test_bIY01W11heYNeFqbIO",
                "metadata": {
                    "credit_reload_amount": "15000"
                }
            }
        },
        {
            "type": "extra-credits-payment-link",
            "id": "plink_1QP4pEPUHhctoJxaEp1D3my!",
            "attributes": {
                "url": "https://buy.stripe.com/test_bIY01W11hesdfaeEasd",
                "metadata": {
                    "credit_reload_amount": "80000"
                }
            }
        }
    ]
}
```